R/safeS3Methods.R

```diff
@@ -106,7 +107,7 @@ print.safeTest <- function (x, digits = getOption("digits"), prefix = "\t",
   }
   cat("\n")
 
-  statValue <- x[["statistic"]]
+  if(!is.null(x[["statistic"]])){statValue <- x[["statistic"]]}
```

Comment:
```
if (!is.null(x[["statistic"]]))
  statValue <- x[["statistic"]]
```

Reply:
Dit heb ik opgelost in de print gedeelte, je kunt gewoon statValue is null hebben, maar dat die bij het printen hier op checked, dus beneden waar je op exists checked vervangen door

```
if (!is.null(statValue))
    out <- c(out, paste(names(statValue), "=", format(statValue, digits = max(1L, digits - 2L))))
```

wat ik dus gedaan heb in mijn code